.github/release-drafter.yml

```diff
@@ -0,0 +1,14 @@
+branches:
```

Comment:
I understand about the release builder, but release drafter is no longer useful since GitHub includes this feature :thinking:

Reply:
That's what I thought too, but Github only allows to get a pre-generated release note when you are creating the release but cannot create a draft with an asset. https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuring-automatically-generated-release-notes